src/js/post-911-gib-status/components/EnrollmentHistory.jsx

```diff
@@ -22,28 +21,38 @@ class EnrollmentHistory extends React.Component {
     */
 
     // History explanation box
+    let historyExplanationBox;
     const benefitMaybePending =
-      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement > 0) &&
-      enrollmentData.usedEntitlement === 0;
+      (enrollmentData.percentageBenefit > 0 || enrollmentData.originalEntitlement.months > 0) &&
+      (enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0);
     const historyMayLookIncorrect = enrollments.length > 0 || benefitMaybePending;
-    const noEnrollmentHistory = enrollments.length === 0 && enrollmentData.usedEntitlement > 0;
-    const historyHeading = noEnrollmentHistory ?
-                           (<h4>You don't have any enrollment history</h4>) :
-                           (<h4>Does something look wrong in your enrollment history?</h4>);
-    const historyExplanation = noEnrollmentHistory ?
-                               (<span>Your enrollment history may not be available if:</span>) :
-                               (<span>Certain enrollments may not be displayed in this history if:</span>);
-    const historyExplanationBox = (historyMayLookIncorrect || noEnrollmentHistory) && (
-      <div className="feature">
-        {historyHeading}
-        {historyExplanation}
-        <ul>
-          <li>Your school made a request to us that's still in process, or</li>
-          <li>You made a request to us that's still in process, or</li>
-          <li>You used or are using your benefit for flight, on-the-job, apprenticeship, or correspondence training</li>
-        </ul>
-      </div>
-    );
+    const noEnrollmentHistory = enrollmentData.usedEntitlement.months === 0 && enrollmentData.usedEntitlement.days === 0;
```

Comment:
This `noEnrollmentHistory` check is meant to cover Scenario B: Applied but haven't used any of their benefit yet. Scenario B, according to the ticket, is based on if months used = 0, so that's why I'm checking `usedEntitlement` is 0, for both months and days, here.